scripts/download-relative-fonts

```diff
@@ -0,0 +1,11 @@
+mkdir bpk-fonts-tmp
```

Comment:
Can this script be executed automatically after the `bundle exec pod update`? https://guides.cocoapods.org/syntax/podfile.html#post_install

Reply:
Yeah - although we'd need to ensure it fails gracefully when SSH access to GitHub fails